src/kaltura-live-plugin.tsx

```diff
@@ -132,19 +138,14 @@ export class KalturaLivePlugin
             this.updateLiveTag();
             this._isActive = true;
             this._player.addEventListener(this._player.Event.FIRST_PLAY, this._handleFirstPlay);
-            this._player.addEventListener(
-                this._player.Event.TIMED_METADATA,
-                this._handleTimedMetadata
-            );
             this._player.configure({
                 plugins: { kava: { tamperAnalyticsHandler: this._tamperAnalyticsHandler } }
             });
             this.updateLiveStatus();
         }
     };
 
-    private _handleTimedMetadata = (e: any) => {
-        this.updateLiveTag();
```

Comment:
why it removed?

Reply:
according code from playkig-js the isOnLive property updated only on SEEKING event, so it does not have sense check isOnLive property after TimedMetadata comes
https://github.com/kaltura/playkit-js/blob/master/src/player.js#L1696-L1700

Reply:
so new implementation updates Live tag only when `SEEKED` event happened